homeassistant/components/dsmr/config_flow.py

```diff
@@ -308,14 +309,14 @@ async def async_validate_dsmr(
         data = input_data
 
         try:
-            is_rfxtrx = False
+            protocol = DSMR_PROTOCOL
```

Comment:
Please move it up out of the try... except block.

Reply:
Changed as requested.

(IMHO it looks worse this way, since the cohesion between assigning the protocol variable and invoking the validation function is lost. To me, it was more clear what is going on when those two lines are adjacent. For the try/except block it makes no real difference.)